components/DrawControls.svelte

```diff
@@ -62,8 +62,6 @@
   let routeSnapper;
 
   onMount(async () => {
-    const map = getMap();
```

Comment:
All the places previously grabbing map from context can now just import from `stores.js` and subscribe as usual. The `$map` syntax is simplest, since that auto-unsubscribes upon component teardown (https://svelte.dev/tutorial/auto-subscriptions)